README.md

```diff
@@ -75,6 +79,12 @@ We provide two pre-trained models:
 
   [![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.3370486.svg)](https://doi.org/10.5281/zenodo.3370486)
 
+Furthermore, we provide a model for speech enhancement trained by [Sony Corporation](link)
+
+* __`umxse`__ speech enhancement model is trained on the 28-speaker version of [Voicebank](https://ieeexplore.ieee.org/document/6709856) + [Demand](https://zenodo.org/record/1227121). 
```

Comment:
* __`umxse`__ speech enhancement model is trained on the 28-speaker version of the [Voicebank+DEMAND corpus](https://datashare.is.ed.ac.uk/handle/10283/1942?show=full).